### ISSUE TITLE
Runs, but won't play music from  /usr/local/bin/

### ISSUE THREAD

**Issue by tylerecouture** [EragonJ/Kaku#443]
Hi!  I can't get Kaku to play music when installed in /usr/local/bin/ (root owner), "Kaku file is 755 (-rwxr-xr-x) on Ubuntu 16.04

I'm trying to install it for all users (with a .desktop file)

Kaku opens and runs, but the video/music just gives the loading swirly image forever. 

Any ideas what I can do?  Permissions to get this working?  It plays music fine if the user downloads and runs it directly.

Thanks!

**Comment by mimi89999** [comment]
Can't you put it in /opt ?

**Comment by tylerecouture** [comment]
It won't play in /opt either.  Kaku opens and lists videos/songs, but they won't play.

**Comment by EragonJ** [comment]
In the next release, i'll release AppImage file and i hope this can fix problems you guys have :)

**Comment by SkyzohKey** [comment]
@tylerecouture here's a script for making .desktop file for Kaku: https://github.com/EragonJ/Kaku/issues/446

**Comment by tylerecouture** [comment]
@SkyzohKey will this make it play videos from /opt or /usr/...  ?

**Comment by EragonJ** [comment]
@tylerecouture 

1. Is there certain reason that you need to put that in that folder ? 
2. If you put that to different folders, does Kaku still work like a charm ?

**Comment by tylerecouture** [comment]
@EragonJ 
1. I want to install in a computer lab of about 35 computers, I don't care at all where it installs to, as long as it's not in /home
2. The only place it seems to work (music/videos play) is if a user installs it  (i.e. plops it) somewhere in their home directory.  This is not feasible because I have hundreds of users and their home drives are networked.  

I just want to be able to install it on each computer, so all users can use it, like any other standard piece of software.  I don't care where it goes.  Do you have a suggestion?

**Comment by mimi89999** [comment]
@tylerecouture I put Kaku in /opt/Kaku/ and it runs without any issues (the latest version from the website). Did you give correct permissions? What is in the shell? (ctrl +shift + i) on GNU/Linux.

**Comment by tylerecouture** [comment]
@mimi89999 Kaku file is 755 (-rwxr-xr-x).  It runs, the program opens and I can search for music and pick songs to play, but no sound comes and no video appears.  I assume this is a permission issue with something somewhere, but no idea what file or where, something to do with the actual streaming.

**Comment by mimi89999** [comment]
Could you open the inspector?

**Comment by tylerecouture** [comment]
Ctrl +Shift+i does nothing for me (Ubuntu 16.04) sorry.

**Comment by tylerecouture** [comment]
Ctrl +Shift+i does nothing for me (Ubuntu 16.04) sorry.

If I run it from the command Line:
`~$ /opt/Kaku-linux64/Kaku` it ... works!  Does that mean there's something wrong with my .desktop file?

**Comment by tylerecouture** [comment]
```
Here was the basci .desktop file I was testing with.  So if I just type Kaku from the launcher/search it would run the program and search songs etc, but not play them:

[Desktop Entry]
Version=1.0
Name=Kaku
Comment=Next generation music player
Exec=/opt/Kaku-linux64/Kaku
Icon=Kaku.png
Terminal=false
Type=Application
Categories=Utility;Application;
```

**Comment by tylerecouture** [comment]
Apologies for all the messages as I try to figure this out.  If I change the desktop file so that `Terminal=true` Kaku works properly and the music/videos will play, but it also opens up a blank Terminal.  I'm not familiar enough with Linux to understand what's going on here, or why that's necessary.

**Comment by EragonJ** [comment]
@tylerecouture can you check whether you can run this binary from your shell ? 

path would be - `Kaku-linux64/Kaku/resources/app.asar.unpacked/node_modules/youtube-dl/bin/youtube-dl`

This is the main thing that we use to help us play tracks.

**Comment by EragonJ** [comment]
I think that may be related to the permission issue ? Try to give it 777 and see whether you can play it without root permission ?

**Comment by mimi89999** [comment]
@tylerecouture You need to put:
Exec=/./opt/Kaku/Kaku

Also, you might have to wait for one minute before the song starts playing.

**Comment by EragonJ** [comment]
Just released 1.8.0 and I think AppImage format should fix all the problems that you guys have, if there is anything new showing up, feel free to open a new issue ! Thanks
